### PR TITLE
Added Support for Aurora Wall Remotes

### PR DESCRIPTION
**Pull request by ashp8i**
Added Support for Aurora AOne One Gang Wireless Battery Rotary Dimmer and Aurora AOne Two Gang Wireless Battery Rotary Dimmer

### CODE REVIEW

**Comment by ashp8i** [comment]
Not sure why lint is failing

File: devices.js [hunk 1]
```diff
@@ -14205,6 +14205,46 @@ const devices = [
             await reporting.instantaneousDemand(endpoint);
         },
     },
+    {
+        zigbeeModel: ['1GBatteryDimmer50AU'],
+        model: 'AU-A1ZBR1GW',
+        vendor: 'Aurora Lighting',
+        description: 'AOne one gang wireless battery rotary dimmer',
+        fromZigbee: [fz.battery, fz.command_on, fz.command_off, fz.command_step, fz.command_step_color_temperature],
+        toZigbee: [tz.light_brightness_step],
```
Comment (Koenkk) [inline devices.js:14214]: Why was `tz.light_brightness_step` added here? Would expect it to be empty. Same for https://github.com/Koenkk/zigbee-herdsman-converters/pull/2032/files#diff-5520dd49755e0a353d3857b8963a91cb4b8d87a01b6b8eb7bb6cce1c180eac83R14233

File: devices.js [hunk 2]
```diff
@@ -14205,6 +14205,46 @@ const devices = [
...
+        exposes: [e.battery(), e.action([
+            'on', 'off', 'brightness_step_up', 'brightness_step_down', 'color_temperature_step_up', 'color_temperature_step_down'])],
+        meta: {configureKey: 1, battery: {voltageToPercentage: '3V_2100'}},
+        endpoint: (device) => {
```
Comment (Koenkk) [inline devices.js:14218]: This shouldn't be needed.

File: devices.js (hunk 1 above)
Comment (ashp8i) [reply devices.js]: sorry not a pro got confused by the controller giving out the step value:

DEBUG: Received Zigbee message from 'Single Rotary Dimmer Remote', type 'commandStep', cluster 'genLevelCtrl', data '{"stepmode":0,"stepsize":26,"transtime":3}' from endpoint 1 with groupID 0
INFO: MQTT publish: topic 'zigbee2mqtt/Single Rotary Dimmer Remote', payload '{"action":"brightness_step_up","action_step_size":26,"action_transition_time":0.03,"linkquality":60,"update":{"state":"idle"},"update_available":false}'

Comment (ashp8i) [reply devices.js]: thought passing on the value might be of some importance

File: devices.js (hunk 2 above)
Comment (ashp8i) [reply devices.js:14218]: which line are you referring to

Comment (ashp8i) [reply devices.js:14218]: the device shows a greenpower: -

image

not sure if it is of any relevance

Comment (ashp8i) [reply devices.js:14218]: also 
image

Comment (ashp8i) [reply devices.js:14218]: when used with an aurora hub which I do have at hand dimmer ready with a zigbee sniffer also supports changing colour and color wheel possibly scenes

Comment (Koenkk) [reply devices.js:14218]: I'm refering to the endpiont line

File: devices.js (hunk 1 above)
Comment (Koenkk) [reply devices.js]: passing = fromZigbee, sending to device = toZigbee

File: devices.js (hunk 2 above)
Comment (ashp8i) [reply devices.js]: I realised it won't be needed as there are only two endpoints1 and 242 and the one being used is 1 any idea what 242 would do (greenpower?)

Comment (Koenkk) [reply devices.js]: Probably this device can serve as a green power sink or proxy. (for more info search Zigbee green power).

**Pull request merged**